src/components/screens/multiSignature/summary/summary.js

```diff
@@ -41,10 +43,31 @@ const Summary = ({
           nonce: account.sequence.nonce,
           senderPublicKey: account.summary.publicKey,
         },
+        isHwSigning,
       }, token),
     );
 
-    if (!error) {
+    if (error) {
+      nextStep({ error });
+    }
+
+    if (isHwSigning) {
```

Comment:
As explained, we should contain tx signature in a utility function, not a visual component.